task-05/NastyaNasalevich/app/js/dashboard/panelOfUsers.js

```diff
@@ -0,0 +1,112 @@
+/* exported panelOfUsers */
+/* global filter */
+/* global sorter */
+/* global operatorsChat */
+/* global removeChildren */
+/* global sendRequestToDatabase */
+
+
+var panelOfUsers = (function createpanelOfUsers() {
+    var userList;
+    var sorterTag;
+    var filterTag;
+    var selectUserId;
+    
+    function addUser(userBody, userId) {
+
+        var userElement = document.createElement('div');
+        var userNameElement = document.createElement('div');
+        var userStatusElement = document.createElement('div');
+        var chatStateElement = document.createElement('div');
+        userNameElement.classList.add('user-name-element');
+        userStatusElement.classList.add('user-status-element');
+        chatStateElement.classList.add('chat-state-element');
+    
+        userNameElement.innerHTML = userBody.userName ? userBody.userName : 'Anonymous';
+    
+        if (userBody.isChatHidden) {
+            chatStateElement.innerHTML = ' - ';
+        } else {
+            chatStateElement.innerHTML = '[ ]';
+        }
+    
+        if (!userBody.isRead) {
+            userNameElement.classList.add('unread-state');
+        }
+        
+        if (new Date() - new Date(userBody.lastMessageDate) <= 600000) {
```

Comment:
https://ru.wikipedia.org/wiki/%D0%9C%D0%B0%D0%B3%D0%B8%D1%87%D0%B5%D1%81%D0%BA%D0%BE%D0%B5_%D1%87%D0%B8%D1%81%D0%BB%D0%BE_(%D0%BF%D1%80%D0%BE%D0%B3%D1%80%D0%B0%D0%BC%D0%BC%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5)